### ISSUE TITLE
Consider supporting rediss:// schema

### ISSUE THREAD

**Issue by MichaelXavier** [informatikr/hedis#141]
I came across this while browsing the docs from the python redis library. It looks like there's actually a [provisional registration](https://www.iana.org/assignments/uri-schemes/prov/rediss) of `rediss://` with the iana as a scheme used for redis over TLS. I wonder if hedis should support this. Considering that it would parse into a `ConnectInfo`, would it just set `connectTLSParams = Just (defaultParamsClient host "")`?

**Comment by k-bx** [comment]
Yeah, I think that might make sense. If anyone's interested -- you're welcome to make a PR!

**Comment by qnikst** [comment]
Rediss:// schema is now fully supported. Add all tests were added.

Support will appear in the next release 0.16.
